tests/unit/test_mysqlsh_helpers.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+import unittest
+from unittest.mock import call, patch
+
+from ops.pebble import ExecError
+
+from mysqlsh_helpers import (
+    MySQL,
+    MySQLAddInstanceToClusterError,
+    MySQLConfigureInstanceError,
+    MySQLConfigureMySQLUsersError,
+    MySQLCreateClusterError,
+)
+
+
+class TestMySQL(unittest.TestCase):
+    def setUp(self):
+        self.mysql = MySQL(
+            "127.0.0.1",
+            "test_cluster",
+            "password",
+            "serverconfig",
+            "serverconfigpassword",
+            "clusteradmin",
+            "clusteradminpassword",
+            None,
+        )
+
+    @patch("mysqlsh_helpers.MySQL._run_mysqlcli_script")
+    def test_configure_mysql_users(self, _run_mysqlcli_script):
+        """Test failed to configuring the MySQL users."""
+        _run_mysqlcli_script.return_value = b""
+
+        _expected_configure_user_commands = " ".join(
+            (
+                "UPDATE mysql.user SET authentication_string=null WHERE User='root' and Host='%';",
```

Comment:
Is there a way to save this as as a file and read it in line by line. Similar to what was done [here](https://github.com/canonical/mongodb-operator/blob/0a66c1ff327faabf6c22ac5246b9b28c14b647b5/tests/unit/test_charm.py#L46-L53)

Reply:
Yes, but tbh i don't feel there's enough of it to justify the overhead.